source/code-generation/query-converter/bulk-conversion.txt

```diff
@@ -0,0 +1,100 @@
+.. _rm-bulk-conversion:
+
+===============
+Bulk Conversion
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can select and convert one or more queries, stored procedures, 
+triggers, and views in bulk by enabling the multi-select toggle. 
+Bulk conversion creates a job that queues and executes each query 
+conversion individually from a single request.
+
+Copy and paste your queries into the query converter individually and 
+then bulk convert a selection to update them to work with MongoDB and
+your migrated schema.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+- .. include:: /includes/fact-query-converter-disclaimer.rst
+
+- .. include:: /includes/fact-view-query-conversion-history.rst
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Navigate to the query converter pane
+
+      From the :guilabel:`Code Generation` tab, click the 
+      :guilabel:`Query Converter` pane.
+
+   .. step:: Open the SQL query converter view
+
+      - If it is your first time using the query converter in your 
+        project, click :guilabel:`Paste SQL Query`.
+      - If you have previously converted SQL code in your project, click the
+        :guilabel:`+ ADD` button on the left-hand pane next to 
+        :guilabel:`Queries`.
+
+   .. step:: Enable multi-selection
+
+      Click the :guilabel:`Enable multi-selection` toggle to show the query 
+      selection pane.
+
+   .. step:: Select your conversions
+
+      - To include or exclude a query from your bulk conversion job,
+        click the :icon-fa5:`check-square` icon next to the field name.
+
+      - To include or exclude entire categories from your download,
+        click :icon-fa5:`check-square` next to the category name.
+   
+   .. step:: Convert and test syntax
+
+      a. Click the :guilabel:`Convert` button. The converted MongoDB 
+         syntax code displays in the 
+         :guilabel:`Converted MongoDB Query` pane.
+
+         .. include:: /includes/fact-query-converter-failure-errors.rst
+
+         .. tip:: 
+
+            To rename or delete queries from your project, click the 
+            :icon-fa4:`ellipsis-h` button on the :guilabel:`SQL Query` 
+            pane and select the :icon-fa5:`pencil-alt` or 
+            :icon-fa4:`trash` icon.
+
+      #. Click the :icon-fa5:`copy` icon to copy the MongoDB syntax 
+         to your clipboard.
+
+      #. Test and validate the generated MongoDB syntax.
```

Comment:
How?

Reply:
Great point! I'm using the direction we provide from a [single query conversion](https://www.mongodb.com/docs/relational-migrator/code-generation/query-converter/convert-queries/#convert-and-test-syntax) and wonder if it's particularly useful here (regardless of content). Do we want to highlight where you select and copy individual completed conversions as well or leave both of these to the single query task?

Reply:
Agree we shouldn't duplicate content from other topics. I'd suggest a "Next steps" section suggesting to review and test the code of each converted query, with a link to the Test Bench section.